LyokoAPI/Events/SectorCreationEvent.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System.Reflection;
+using LyokoAPI.API;
+using LyokoAPI.VirtualStructures;
+using LyokoAPI.VirtualStructures.Interfaces;
+
+namespace LyokoAPI.Events
+{
+    public class SectorCreationEvent
+    {
+        private static event Events.OnSectorEvent SectorE;
+        public static void Call(ISector sector)
```

Comment:
There should also be a way of calling this with a string. See the tower activation event for examples.